docs/quick_tutorial/index.rst

```diff
@@ -4,8 +4,7 @@
 Quick Tutorial for Pyramid
 ==========================
 
-Pyramid is a web framework for Python 2 and 3. This tutorial gives a Python
-3/2-compatible, high-level tour of the major features.
+Pyramid is a web framework for Python 3. This tutorial gives a Python 3-compatible, high-level tour of the major features.
```

Comment:
This tutorial gives a high-level tour of the major features.

Reply:
Fixed in ae9453732bcbd4a9812e6cd9232bba6d469a9953